num/dualquat/dual_fike.go

```diff
@@ -0,0 +1,152 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Derived from code by Jeffrey A. Fike at http://adl.stanford.edu/hyperdual/
+
+// The MIT License (MIT)
```

Comment:
Why does this need a license? It seems like this code is yours using the other code that is related to Jeff's

Reply:
The is very similar to the code from Jeff's for the dual reals. The dual complex is adequately different and I removed the distinction there. I don't mind either way, but if we do remove it, I'd prefer to check with him first. You know him, so do you want to ask?

Reply:
It's okay to leave in, I looked at the other code and it's more similar than I remembered.